### ISSUE TITLE
Add queryResults to the viewer

### ISSUE THREAD

**Issue by filiphric** [metabase/metabase#50618]
queryResults were not in the scope of the original debugger, this probably needs to be built with an interface that can help navigate through it easily. We can still show the raw data, but a viewer would remove obstacles towards smooth debugging

**Comment by filiphric** [comment]
@thebiglabasky I think actually some basic design would help here. I’m not sure at this moment what would be the best way to display queryResults data. We always show the raw searchable data, but if there’s a UI that could improve the debugging experience I don’t know what that would look like

**Comment by thebiglabasky** [comment]
True. I think we could start simply showing the raw JSON data. and leave folks navigate it using the editor as that's the simplest to implement.

We can later imagine if we want more of a table view of sorts since most results are tabular by nature, but I don't think that's a priority until we have had a few people using it.
I'll clarify this issue and create another in the backlog to have a fancier results view
